server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/task/SqlPlanTask.java

```diff
@@ -426,11 +431,10 @@ private String uploadToOSS(String filePath) {
             try {
                 String objectName = cloudObjectStorageService.upload(file.getName(), file);
                 ossAddress = String.valueOf(cloudObjectStorageService.generateDownloadUrl(objectName));
-                log.info("upload sql plan task result file to OSS, file name={}", file.getName());
+                log.info("succeed to upload sql plan task result file to OSS, file download url = {}", ossAddress);
```

Comment:
seems syntax error, may 

`Upload sql plan task result to cloud object storage successfully, objectName={}`

and, downloadUrl contains ak/sk/sts, we may not output it in log content.

so as below log output.

Reply:
Have been changed.